### PR TITLE
Update translations for Gaphor

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Gaphor/Gaphor](https://hosted.weblate.org/projects/gaphor/gaphor/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by amolenaar** [comment]
Thanks @stansz84.

I fixed the placeholders (e.g. `{name}`). Those should not be translated.

File: po/fr.po [hunk 1]
```diff
@@ -2897,9 +2831,8 @@ msgid "New _Diagram"
 msgstr "Nouveau _Diagramme"
 
 #: gaphor/ui/modelbrowser.py:592
-#, fuzzy
 msgid "New _Element"
-msgstr "Nouveau composant"
+msgstr "Nouvel élément"
```
[the remark this answers is not in the document]
Comment (stansz84) [reply po/fr.po:2835]: To use the shortcut, the correct answer is "Nouvel _Elément"

[the remark this answers is not in the document]
Comment (danyeaw) [reply po/fr.po]: Thanks!

**Comment by danyeaw** [comment]
@all-cobtributits please add @stansz84 for translation

**Comment by danyeaw** [comment]
@all-contributors please add @stansz84 for translation

**Pull request merged**